recipes/citation-compass/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python_min:
```

Comment:
conda-forge does not support python<3.9. I would strongly advise removing this and just letting it be set by our global pinning:
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L828-L831